lib/scrum_lint/interactive_linters/card/missing_checklist_item.rb

```diff
@@ -63,7 +63,7 @@ def lookup_project_card(context_link)
         ScrumLint::Trello::CardMapper.(
           trello_card,
           list: trello_card.list,
-          board_name: trello_card.board.name,
+          board_name: trello_card.board.name
```

Comment:
RuboCop yelled: `Style/TrailingCommaInArguments`

Reply:
so is the idea for this PR just to get everything passing, and then if we want to change the rubocop rules we can go back and do that in followups?

Reply:
@kaifius Yup, exactly. I wanted to touch the minimum amount of things to get a passing build and then be more intentional in follow-ups. If you have some suggestions for things to improve, let me know!

Reply:
cool, yeah, i think we should enable this rule and rename `get_value` to enable the other rule in followups. Or maybe even paste in synchroform's rubocop.yml and see what fails